core/src/commonMain/kotlin/entity/Member.kt

```diff
@@ -93,6 +93,12 @@ public class Member(
     /** The [GuildMemberFlags] of this member. */
     public val flags: GuildMemberFlags get() = memberData.flags
 
+    /**
+     * The [Permissions] of this member.
+     * Non-null in interaction contexts.
+     */
```

Comment:
```suggestion
    /**
     * The total [Permissions] of this member in the channel an interaction was sent from.
     *
     * This is only non-null when obtained from a [GuildInteraction][GuildInteraction.user].
     */
```
slightly adjusted documentation, see [here](https://discord.com/developers/docs/resources/guild#guild-member-object-guild-member-structure)

this would also need an import for `dev.kord.core.entity.interaction.GuildInteraction` to resolve the reference correctly

Reply:
awesome, thank you! will make the changes now